etc/config/csharp.amazon.properties

```diff
@@ -2,16 +2,30 @@ compilers=&csharp
 supportsBinary=false
 needsMulti=false
 compilerType=csharp
-defaultCompiler=dotnet601csharp
+defaultCompiler=dotnet700csharp
 
-group.csharp.compilers=dotnet601csharp
+group.csharp.compilers=dotnet601csharp:dotnet6011csharp:dotnet700csharp
 
 compiler.dotnet601csharp.exe=/opt/compiler-explorer/dotnet-v6.0.1/bin/coreclr/Linux.x64.Checked/.dotnet/dotnet
 compiler.dotnet601csharp.name=.NET 6.0.101
 compiler.dotnet601csharp.clrDir=/opt/compiler-explorer/dotnet-v6.0.1/bin/coreclr/Linux.x64.Checked
-compiler.dotnet601csharp.runtimeId=linux-x64
 compiler.dotnet601csharp.targetFramework=net6.0
+compiler.dotnet601csharp.bclVersion=6.0.0
 compiler.dotnet601csharp.buildConfig=Release
-compiler.dotnet601csharp.additionalSources=
 compiler.dotnet601csharp.langVersion=latest
-compiler.dotnet601csharp.nugetPackages=/opt/compiler-explorer/dotnet_nuget-v6.0.0/packages
+
+compiler.dotnet6011csharp.exe=/opt/compiler-explorer/dotnet-v6.0.11/bin/coreclr/Linux.x64.Checked/.dotnet/dotnet
+compiler.dotnet6011csharp.name=.NET 6.0.403
+compiler.dotnet6011csharp.clrDir=/opt/compiler-explorer/dotnet-v6.0.11/bin/coreclr/Linux.x64.Checked
+compiler.dotnet6011csharp.targetFramework=net6.0
+compiler.dotnet6011csharp.bclVersion=6.0.10
+compiler.dotnet6011csharp.buildConfig=Release
+compiler.dotnet6011csharp.langVersion=latest
+
+compiler.dotnet700csharp.exe=/opt/compiler-explorer/dotnet-v7.0.0/bin/coreclr/Linux.x64.Checked/.dotnet/dotnet
+compiler.dotnet700csharp.name=.NET 7.0.100
+compiler.dotnet700csharp.clrDir=/opt/compiler-explorer/dotnet-v7.0.0/bin/coreclr/Linux.x64.Checked
+compiler.dotnet700csharp.targetFramework=net7.0
+compiler.dotnet700csharp.bclVersion=7.0.0-rc.1.22423.16
```

Comment:
This version should match the name of the directory in `/opt/compiler-explorer/dotnet-v7.0.0/bin/coreclr/Linux.x64.Checked/.dotnet/shared/Microsoft.NETCore.App`.